Sources/Camera/ZLCustomCamera.swift

```diff
@@ -467,6 +467,7 @@ open class ZLCustomCamera: UIViewController {
             recordVideoPlayerLayer?.backgroundColor = UIColor.black.cgColor
             recordVideoPlayerLayer?.videoGravity = .resizeAspect
             recordVideoPlayerLayer?.isHidden = true
+            cameraConfig.overlayView?.isHidden = false
```

Comment:
Why add 'cameraConfig.overlayView?.isHidden = false' here?

Reply:
`isHidden` might be explicitly specified as `true` on the user side; I will double-check, but it might indeed be redundant. Thanks!

Reply:
Removed it as it's not needed by default, thanks!